plugins/org.eclipse.n4js.regex/build.properties

```diff
@@ -6,3 +6,4 @@ bin.includes = model/,\
                .,\
                plugin.xml,\
                about.html
+additional.bundles = com.ibm.icu
```

Comment:
why is this necessary?

Reply:
Due to newer EMF version?

Reply:
com.ibm.icu is required to generate the language infrastructure. Using this mechanism here ensures that we do not introduce a real code dep. by accident